tests/models/test_semantic_segmentation.py

```diff
@@ -44,11 +45,15 @@
     SemanticSegmentor,
     WSIStreamDataset,
 )
+from tiatoolbox.utils import env_detection as toolbox_env
 from tiatoolbox.utils.misc import imread, imwrite
 from tiatoolbox.wsicore.wsireader import get_wsireader
 
-ON_TRAVIS = True
-ON_GPU = not ON_TRAVIS and torch.cuda.is_available()
+ON_GPU = toolbox_env.has_gpu()
```

Comment:
this variable is not used very much. I would prefer to skip it or use it everywhere.

Reply:
I dont understand. `ON_GPU` is used at 14 different places in the files.

Reply:
you are still using `toolbox_env.has_gpu()` within the file. I am suggesting to use the variable everywhere or just not use it at all. just to be consistent.

Reply:
Fixed in https://github.com/TissueImageAnalytics/tiatoolbox/pull/258/commits/9260d9c2d654ac6f097cce2e8468e9c24fccb7ad